components/FrontPage/topbar.jsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import React from "react";
+import Btn from "./button";
+import ThemeSwitcher from "./themeSwitcher";
+import { useState, useEffect } from "react";
+
+export default function TopBar() {
+  // Logic for changing the scrollbar whenever it's not at the very top
+  const [isAtTop, setIsAtTop] = useState(true);
+  useEffect(() => {
+    const handleScroll = () => {
+      const scrollTop =
+        window.pageYOffset || document.documentElement.scrollTop;
+      setIsAtTop(scrollTop === 0);
+    };
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+  //TODO: figure out a better way to scale the topbar, currently it just hides the middle section
+  return (
+    <nav
+      className="fixed left-1 right-1 top-1 z-30 grid h-[63px] grid-cols-2 rounded-full border-midnight-400 bg-chalk-50 px-[20px] transition-all dark:border-dusk-50 dark:bg-midnight-300 sm:grid-cols-3"
```

Comment:
The design for this is a bit off. Please refer to the previous version's navbar for design. The padding must be correct